lib/fathead/sqlalchemy/output.txt

```diff
@@ -1,405 +1,405 @@
+interfaces.PropComparator.any_	A										<section class="prog__container"><pre><code>any_()</code></pre><p>\ninherited from the any_() method of ColumnOperatorsProduce a any_() clause against the\nparent object.\nNew in version 1.1.\n</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.interfaces.PropComparator.any_
+MultipleResultsFound	A										<section class="prog__container"><pre><code>exception sqlalchemy.orm.exc.MultipleResultsFound</code></pre><p>A single database result was required but more than one were found.</p></section>	http://docs.sqlalchemy.org/en/latest/orm/exceptions.html#sqlalchemy.orm.exc.MultipleResultsFound
+items	A										<section class="prog__container"><pre><code>items() &#8594; list of D's (key, value) pairs, as 2-tuples</code></pre><p>\ninherited from the items() method of dict</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.instrumentation.ClassManager.items
+attributes.QueryableAttribute.match	A										<section class="prog__container"><pre><code>match(other, **kwargs)</code></pre><p>\ninherited from the match() method of ColumnOperatorsImplements a database-specific &#8216;match&#8217; operator.match() attempts to resolve to\na MATCH-like function or operator provided by the backend.\nExamples include:\nPostgreSQL - renders x @@ to_tsquery(y)\nMySQL - renders MATCH (x) AGAINST (y IN BOOLEAN MODE)\nOracle - renders CONTAINS(x, y)\nother backends may provide special implementations.\nBackends without any special implementation will emit\nthe operator as &#8220;MATCH&#8221;.  This is compatible with SQlite, for\nexample.\n</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.attributes.QueryableAttribute.match
+InspectionAttr	A										<section class="prog__container"><pre><code
```

Comment:
@shravan97 looking at this close, I see that you're converting a lot of different kind of markup into plain text, and the result here is pretty hard to read.

If you pull out `<p>` and `<ul>` from the source and escape their contents, you should be able to preserve the same layout we see in the documentation.

As well, I think you should be parsing the `Parameters` separately. into a `<ul>` with a proper subtitle, or ignoring it for now...